src/Tasks/Common/Resources/Strings.resx

```diff
@@ -653,4 +653,8 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1137: It is no longer necessary to use the Microsoft.NET.Sdk.WindowsDesktop SDK. Microsoft.NET.Sdk can be used instead.</value>
     <comment>{StrBegin="NETSDK1137: "}</comment>
   </data>
+  <data name="InvalidWindowsTargetPlatformVersion" xml:space="preserve">
+    <value>NETSDK1138: {0} is not a valid Windows TargetPlatformVersion.</value>
```

Comment:
I think we should make this error message (and the logic to raise it) generic to different target platforms:

> NETSDK1138: {0} is not a valid TargetPlatformVersion for {1}.  Valid versions include: \r\n{2}

Where {2} is a newline-separated list of the valid target platform versions

Also this will need to change to NETSDK1139 or later once #12202 is merged.